_component_design/tables-controls.md

```diff
@@ -152,27 +152,28 @@ Apply sorting to table headers via a click interaction and an icon visual indica
 
 <section class="static-section" markdown="1">
 
+## Table controls - below
+{: id="below"}
+
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-## Table controls - below
-{: id="below"}
+The following controls can be located at the bottom of a table:
 
-The following controls can be found on the bottom of a table:
-
-- Primary actions
-- Secondary action
-- Pagination actions
-- Select actions
-- Number items displayed actions
+- Rows shown, and total rows
```

Comment:
Minor nit: Is there a reason we're using `-` for bullets here and then `*` on lines 193 on?

Reply:
nope -- no reason.  will fix

Reply:
fixed PR comments.  Feel free to squash and merge when you review this.